### ISSUE TITLE
Specify that 1.3.39 fixes an image error 

### ISSUE THREAD

**Issue by JuaniRios** [paritytech/zombienet#794]
When using 1.3.38, and trying to spawn a native network of a parachain and a relay chain:
```sh
 ~/Downloads/zombienet-linux-x64 spawn -p native scripts/zombienet/native/base-rococo-local.toml
```
 I get:
` Error:          Error: Image's URL is invalid:  parity/polkadot-parachain:latest`

Using 1.3.39 solves the error.

.toml:
```toml
[relaychain]
default_command = "polkadot"
chain = "rococo-local"

	[[relaychain.nodes]]
	name = "alice"
	validator = true

	[[relaychain.nodes]]
	name = "bob"
	validator = true

[[parachains]]
id = 2000
cumulus_based = true
	[[parachains.collators]]
	name = "collator1"
	command = "./target/release/polimec-parachain-node"
	validotor = true
	args = ["-lparachain=debug"]
```

**Comment by wirednkod** [comment]
Thank you @JuaniRios .
We are aware of this, and that is why the 1.3.39 was released (see #780 )

**Comment by wirednkod** [comment]
Based on the ChangeLog it is a `typo fix` that was creating issues on 1.3.38 and that is why version 1.3.39 was released instantly once identified. 

I am not sure what you mean by: 
> it should be more clear on the log when something was seriously broken and the next release fixed it?

Since this is an `Under Active Development` project, developers should always keep updated to the latest release, since many bug/typos/fixes may be introduced between releases.

If there is any more specific suggestion that you could make, other than the existing changelog that denotes the changes between releases, in order to save time to developers, please provide the proposal of solution. 
PR contributions are always welcome.

**Comment by pepoviola** [comment]
Hi @JuaniRios, thanks for the feedback! At the moment we are only including the `auto-generated` release text but improve the `docs` on each release is in our backlog.

Thanks!!
